CHANGELOG.rst

```diff
@@ -7,6 +7,9 @@ This file keeps track of all notable changes to the Cluster Agent.
 Unreleased
 ----------
 
+* Removed job-script parser to extract the name and values of the SBATCH parameters contained in the file.
+* Added logic to pull `execution_parameters` from job submissions and use they as job properties when jobs are submitted.
```

Comment:
```suggestion
* Added logic to pull `execution_parameters` from job submissions and use them as job properties when jobs are submitted.
```

Reply:
solved in [8fa4e05](https://github.com/omnivector-solutions/cluster-agent/pull/60/commits/8fa4e0580f4941a7c65cd33bf59fe91333853929)